09:45:58.272191 INFO [otestpointd/application] commandline: otestpointd -d -l 3 -f /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/log/otestpointd2.log --pidfile /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/run/otestpointd2.pid --uuidfile /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/run/otestpointd2.uuid host-otestpointd2.xml 
09:45:58.272209 INFO [otestpointd/application] uuid: b03f5c34-1452-4822-909a-8bf57b5436d5
09:45:58.305253 INFO [otestpointd] discovery service listening on tcp://127.0.0.1:8882
09:45:59.567141 INFO [otestpoint-probe/node-2/0/manager] available probes: EMANE.PhysicalLayer.Counters.General EMANE.PhysicalLayer.Tables.Events EMANE.PhysicalLayer.Tables.Status
09:45:59.591917 INFO [otestpoint-probe/node-2/1/manager] available probes: EMANE.RFPipe.Counters.General EMANE.RFPipe.Tables.Events EMANE.RFPipe.Tables.Neighbor EMANE.RFPipe.Tables.Status
09:45:59.608754 ERROR [otestpointd/node-2/1/container] Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/otestpoint/emane/probebase.py", line 222, in start
    raise ProbeException("unable to determine build id")

ProbeException: unable to determine build id

09:55:46.641435 INFO [otestpointd/application] shutdown
